README.md

```diff
@@ -895,6 +895,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Swapna Devi](https://github.com/Swapna804)
   - [Syed Sibteali Baqar](https://github.com/sibteali786)
   - [syedareehaquasar](https://github.com/syedareehaquasar)
+  - - [SajadHussain](https://github.com/sajjadcodes)
```

Comment:
```diff
- [wrong]      - - [SajadHussain](https://github.com/sajjadcodes)
+ [Correct]    - [Sajad Hussain](https://github.com/sajjadcodes)
```
Duplicate of #1063 
what about this old PR? https://github.com/EddieHubCommunity/hacktoberfest-practice/pull/1063